client/my-sites/email/email-management/style.scss

```diff
@@ -233,7 +233,9 @@
 		margin-right: 8px;
 	}
 
-	&:hover {
+	&.is-selected,
+	&:hover,
+	&:focus {
```

Comment:
This is a minor additional change that I snuck in here. Here are two screenshots that illustrate the difference it makes when the action menu is initially opened.

<img width="1442" alt="before" src="https://user-images.githubusercontent.com/1101677/171820914-66d5b298-a393-41c4-adfe-839fb8328e55.png">

<img width="1442" alt="after" src="https://user-images.githubusercontent.com/1101677/171820934-e7c276ae-eed0-4768-b00f-2105e59c7078.png">